app/src/main/java/org/jellyfin/mobile/AppPreferences.kt

```diff
@@ -18,6 +18,22 @@ class AppPreferences(context: Context) {
             }
         }
 
+    var instanceAccessToken: String?
+        get() = sharedPreferences.getString(Constants.PREF_INSTANCE_ACCESS_TOKEN, null)
+        set(value) {
+            sharedPreferences.edit {
+                if (value != null) putString(Constants.PREF_INSTANCE_ACCESS_TOKEN, value) else remove(Constants.PREF_INSTANCE_ACCESS_TOKEN)
+            }
+        }
+
+    var instanceUserId: String?
+        get() = sharedPreferences.getString(Constants.PREF_INSTANCE_USER_ID, null)
+        set(value) {
+            sharedPreferences.edit {
+                if (value != null) putString(Constants.PREF_INSTANCE_USER_ID, value) else remove(Constants.PREF_INSTANCE_USER_ID)
+            }
+        }
+
```

Comment:
I'm not sure whether I want to save those into SharedPreferences.. I've already integrated a database for server management in the `compose` branch, and I think I'll cherry-pick that very soon. If it's ok for you, I'd wait for that and refactor your credential persistence to use the new database then.

Reply:
Sounds good to me. Let me know when that's available.